z390test/src/test/groovy/org/z390/test/RunAsmTests.groovy

```diff
@@ -61,6 +61,32 @@ class RunAsmTests extends z390Test {
         assert rc == 0
     }
     @Test
+    void test_TESTFPC1() {
+        env.put('RT1OUT', basePath('tests', 'TESTFPC1.TST'))
+        int rc = this.asmlg(basePath("tests", "TESTFPC1"), *options, "SYSOBJ(+${basePath("linklib")})", 'optable(z390) notrace')
+        this.printOutput()
+        assert rc == 0
+        // Load files to fileData
+        loadFile(basePath("tests", "TESTFPC1.TF1"), 'TF1')
+        loadFile(basePath("tests", "TESTFPC1.TST"), 'TST')
+        // Check files equal
+        assert fileData.get('TF1') == fileData.get('TST')
+        assert rc == 0
```

Comment:
Do you need both asserts? Just wondering. Okay as is. Will wait to hear your response.

Reply:
no, the second one is an oversight. I'll fix that.